hudi-common/src/main/java/org/apache/hudi/common/table/log/block/HoodieLogBlock.java

```diff
@@ -258,14 +257,6 @@ protected void deflate() {
    * @throws IOException -
    */
   private static void safeSeek(FSDataInputStream inputStream, long pos) throws IOException {
```

Comment:
do we still need this method then? just call `inputStream.seek()`?

Reply:
yes, we could remove this. will fix it along with any other feedback which could be given.